### PR TITLE
FixChineseTranslation

### PR DESCRIPTION
**Pull request by lovejoy**
**What this PR does / why we need it**:

fixChineseTranslation

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lovejoy** [comment]
@zjj2wry seems you are the last commiter

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -2390,7 +2390,7 @@ msgstr "显示一个或更多 resources"
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/current_context.go#L48
 #: pkg/kubectl/cmd/config/current_context.go:49
 msgid "Displays the current-context"
-msgstr "显示 current_context"
+msgstr "显示当前 context"
```
Comment (wgliang) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: If you want to modify it, you can change it as `显示当前的 context`

Comment (lovejoy) [reply translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 我想到了来着 但是感觉这个的加的有点怪你不觉得吗 尤其你看上下 都没有的

**Comment by zjj2wry** [comment]
/ok-to-test
if ci failed, please execute hack/update-translations.sh thanks～

**Comment by lovejoy** [comment]
@@zjj2wry  all test is finished

**Comment by lovejoy** [comment]
according to the notice , seems somebody need to approve this pr ? so who is the somebody i should ping ?

**Comment by dims** [comment]
@lovejoy one of these folks should be able to approve - https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES#L19-L31

**Comment by lovejoy** [comment]
/retest

poor guy

**Comment by lovejoy** [comment]
ping @@seans3

**Review by adohe-zz — APPROVED** [review]

**Pull request merged**